CHANGELOG.md

```diff
@@ -73,6 +75,7 @@ Formatted as described on [https://keepachangelog.com](https://keepachangelog.co
 - Empty Python project directory structure
 - Added symlink based data files copier
 
-[Unreleased]: https://github.com/eWaterCycle/ewatercycle/compare/1.0.0...HEAD
+[Unreleased]: https://github.com/eWaterCycle/ewatercycle/compare/1.1.0...HEAD
+[1.1.0]: https://github.com/eWaterCycle/ewatercycle/compare/1.0.0...1.1.0
```

Comment:
This doesn't work yet, but I guess it will once the tag for 1.1.0 is created.

Reply:
yep, this should work after release